### ISSUE TITLE
Game wont read the saves anymore

### ISSUE THREAD

**Issue by SrVolk** [Razzmatazzz/RemnantSaveGuardian#178]
probably me just being really dumb... but my original saves are still there in the original save folder but now the game opens on character creation coz theres no saves.

**Comment by SrVolk** [comment]
welp i managed to fixed it by deleting both sav files from my original saves and changint the first profile.bak to profile.sav

still from what i understand what this app is supposed to do, really weird it would cause this kind of problem.

**Comment by SrVolk** [comment]
and now, i disabled the auto backup just to avoid problems... but the world analyzer wont work on the save lol

**Comment by Razzmatazzz** [comment]
It'd hard to follow what's going on here. RSG doesn't touch your active profile.sav or profile.bak files unless you tell it to restore a previous save. profile.bak isn't the backup that RSG makes; it's a backup the game itself makes.k
